applications/user-generated-content.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Machine translation for user-generated content
+---
+
+# User-Generated Content
+
+Machine translation for **user-generated content (UGC)** is used mainly for **gisting**.
+User-generated content is a good fit for machine translation because it is very high-scale.
+User-generated content is challenging for machine translation because it is open-domain and noisy.
+
+
```

Comment:
### Content types

[Posts and comments](applications/social-networks)
[Live chat](applications/live-chat)
[Product titles, descriptions and reviews](applications/commerce-and-marketplaces)
[In-game content](applications/gaming)

Reply:
This should probably be at the bottom though, below *Goals*